meson.build

```diff
@@ -84,7 +82,6 @@ endif
 
 butterfly_dependencies = [
   blas_dep,
-  lapacke_dep,
```

Comment:
Remove `blas_dep` and `lapacke_dep` from `butterfly_dependencies`. Then, after `butterfly_dependencies` is defined, add:
```
if host_machine.system() == 'linux'
  butterfly_dependencies += [blas_dep, lapacke_dep]
elif host_machine.system() == 'darwin'
  butterfly_dependencies += [blas_dep]
elif host_machine.system() == 'windows'
  error('Windows unsupported currently')
endif
```